fivox/vsdLoader.h

```diff
@@ -48,10 +48,55 @@ class VSDLoader : public EventSource
 
     /**
     * Set the attenuation curve that will be applied to the computed events
+    *
     * @param curve The attenuation curve to apply
     */
     void setCurve( const AttenuationCurve& curve );
 
+    /**
+     * Enable the interpolation of the attenuation values from the dye curve
+     *
+     * @param interpolate true to interpolate the attenuation values before
+     * applying them; false to use the attenuation values as they are in the
+     * dye curve file
+     */
+    void setInterpolation( bool interpolate );
+
+    /**
+     * @return the bounding box of the loaded somas
+     */
+    AABBf getBoundingBoxSomas() const;
+
+    /**
+    * Set the resting potential that will be used for the computation of the VSD
+    *
+    * @param v0 resting potential in mV
+    */
+    void setV0( float v0 );
+
+    /**
+    * Set the multiplier for surface area in background fluorescence term value
+    *
+    * @param g0 multiplier for surface area in background fluorescence term
+    * value
+    */
+    void setG0( float g0 );
+
+    /**
+     * Enable the spike filtering
+     * @param enable true to enable spike filtering; false to use the input
+     * voltages as they are
+     */
+    void setSpikeFilter( bool enable );
+
+    /**
+     * When the spike filtering is enabled, use this to set the action potential
+     * threshold, in millivolts
+     *
+     * @param apThreshold action potential threshold, in millivolts
```

Comment:
The spike filtering is an option used to perform experiments, by changing the action potential threshold. It basically clamps the input voltage to the specified value. This kind of things will be detailed in a brief user guide that I will add in a different PR.